### PR TITLE
add lesson 19 notes

### PR DESCRIPTION
**Pull request by angelsvirkov**
@impactmass , let's use this PR to merge the final exercise notes. I would be adding my exercises here. You could also maybe create a PR to this branch and we can thus have two PRs to review each others notes

### CODE REVIEW

File: lesson19/exercises/overflow/solution/main.css [hunk 1]
```diff
@@ -0,0 +1,200 @@
+body {
```
Comment (impactmass) [inline lesson19/exercises/overflow/solution/main.css:1]: I would like that you write out the styles that are not present in the template at the bottom of the file. That will help me know the exact styles that are changing. Just like you've done for the other exercise.

Comment (angelsvirkov) [reply lesson19/exercises/overflow/solution/main.css:1]: Make sense!. Will do it

Comment (angelsvirkov) [reply lesson19/exercises/overflow/solution/main.css:1]: Done. Please take a look to see if you agree with it

Comment (impactmass) [reply lesson19/exercises/overflow/solution/main.css:1]: Thank you. LGTM.

File: lesson19/exercises/position-redi-logo/template/index.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        I am highly-motivated, result oriented, self-driven, hard-working.
+      </p>
+      <p>
+        This is my home on the Internet. Feel free to browse around, and drop
+        me a line 😎
+      </p>
+    </div>
+    <!-- Part 1 Start: Adjust HTML here -->
+    <img class="photo" src="./photo.png" alt="A photo of me enjoying sunset at the beach">
+    <img src="./redi_logo.png" alt="The logo of ReDI" />
```
Comment (impactmass) [inline lesson19/exercises/position-redi-logo/template/index.html:84]: I just noticed that there is something wrong with the redi_logo image. It is not showing in my Codesandbox, see screenshot. I have re-recreated the CodeSandbox twice and still the same.

Cursor_and_objective-grothendieck-4irkp_-_CodeSandbox

Does this happen for you? please check before we merge.

Comment (angelsvirkov) [reply lesson19/exercises/position-redi-logo/template/index.html:84]: Hmm good catch. It happens yes in Codesandbox. Still don't know why

Comment (impactmass) [reply lesson19/exercises/position-redi-logo/template/index.html:84]: Yea, weird. It looks like during the import into CodeSanbox servers, the image is wrongly marked as text. No idea why. But it shows text in dev tools. Maybe try creating a new image?
Cursor_and_blissful-snyder-ie171_-_CodeSandbox

Comment (angelsvirkov) [reply lesson19/exercises/position-redi-logo/template/index.html:84]: Thanks I would fix it.

**Review by impactmass — APPROVED** [review]

File: lesson19/exercises/position-redi-logo/template/index.html (hunk 2 above)
Comment (impactmass) [reply lesson19/exercises/position-redi-logo/template/index.html]: Thanks

**Pull request merged**